### PR TITLE
added proper folder 

### PR DESCRIPTION
**Pull request by tanwi1234**

### CODE REVIEW

File: 3]. Competitive Programming/04]. GeeksforGeeks/1]. Data Structures/01) hashing/01)intersection of two arrays.cpp [hunk 1]
```diff
@@ -0,0 +1,61 @@
+int NumberofElementsInIntersection (int a[], int b[], int n, int m )
+    {
+        // Your code goes here
+        map<int,int>m;
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/04]. GeeksforGeeks/1]. Data Structures/01) hashing/01)intersection of two arrays.cpp:4]: check this solution again

File: 5]. Projects/Machine Learning & Data Science  (ML-DS)/Python/Deep Learning Projects/Computer Vision/003]. Advanced Real Time Hand-Pose Detection/Advanced Real Time Hand-Pose Detection.ipynb [hunk 2]
```diff
@@ -1,2308 +0,0 @@
-{
```
Comment (AkashSingh3031) [inline 5]. Projects/Machine Learning & Data Science  (ML-DS)/Python/Deep Learning Projects/Computer Vision/003]. Advanced Real Time Hand-Pose Detection/Advanced Real Time Hand-Pose Detection.ipynb:1]: Don't delete files

**Review by AkashSingh3031 — CHANGES_REQUESTED** [review]
Check your 1st Hashing code
And don't delete the files

Try to create a difficulty folder in GFG
image

put correct numbering 
image
if hashing at position 8 then write `08]. Hashing/<Difficulty Level>/<code name with number>`

like `08]. Hashing/2]. Basic/_02)_Unique_Numbers.cpp`
image

**Comment by tanwi1234** [comment]
> Check your 1st Hashing code
> And don't delete the files
> 
> Try to create a difficulty folder in GFG
> image
> 
> put correct numbering
> image
> if hashing at position 8 then write `08]. Hashing/<Difficulty Level>/<code name with number>`
> 
> like `08]. Hashing/2]. Basic/_02)_Unique_Numbers.cpp`
> image

I made this particular  question and its numbering according to cheatcode sheet  provided by gfg.

**Comment by AkashSingh3031** [comment]
> > Check your 1st Hashing code
> > And don't delete the files
> > Try to create a difficulty folder in GFG
> > image
> > put correct numbering
> > image
> > if hashing at position 8 then write `08]. Hashing/<Difficulty Level>/<code name with number>`
> > like `08]. Hashing/2]. Basic/_02)_Unique_Numbers.cpp`
> > image
> 
> I made this particular question and its numbering according to cheatcode sheet provided by gfg.

Try to numbering like this 
`08]. Hashing/2]. Basic/_02)_Unique_Numbers.cpp`

**Comment by AkashSingh3031** [comment]
I close these PRs and again send PRs
Because many files delete by you

**Pull request closed without merging**